docs/components/xr-mode-ui.md

```diff
@@ -19,25 +19,52 @@ to the [`<a-scene>` element][scene]. If we wish to simply toggle the UI, use CSS
 
 ## Properties
 
-| Property      | Description                                                         | Default Value |
-|-----------------------|---------------------------------------------------------------------|---------------|
-| cardboardModeEnabled  | Enables the now deprecated cardboard mode.                          | false         |
-| enabled               | Whether or not to display UI related to entering VR.                | true          |
-| enterVRButton         | Selector to a custom VR button. On click, the button will enter VR. | ''            |
-| enterVREnabled        | If the VR button is displayed when applicable                       | false         |
-| enterARButton         | Selector to a custom AR button. On click, the button will enter AR. | ''            |
-| enterAREnabled        | If the AR button is displayed when applicable                       | false         |
-| XRMode                | If the AR, VR button or both will be displayed.  One of 'ar', 'vr' or 'xr'.| vr            |
+| Property                | Description                                                         | Default Value | One of |
+|-------------------------|---------------------------------------------------------------------|---------------|--------|
```

Comment:
Adding a One Of column makes docs inconsistent with other components

Reply:
Thanks. Removed One Of column from pull request.